yield_plugin/include/yield_plugin/yield_plugin_node.hpp

```diff
@@ -74,8 +74,8 @@ class YieldPluginNode : public carma_guidance_plugins::TacticalPlugin
     // Subscribers
     carma_ros2_utils::SubPtr<carma_v2x_msgs::msg::MobilityRequest> mob_request_sub_;
     carma_ros2_utils::SubPtr<carma_v2x_msgs::msg::BSM> bsm_sub_;
+    carma_ros2_utils::SubPtr<carma_perception_msgs::msg::ExternalObjectList> external_objects_sub_;
     carma_ros2_utils::SubPtr<std_msgs::msg::String> georeference_sub_;
```

Comment:
I think the reason we have the SubPtr is because the publisher is not `rclcpp::Publisher` but `std::shared_ptr<rclcpp_lifecycle::LifecyclePublisher<T>>;` which is `carma_ros2_utils::PubPtr<T>`. So it might better to be consistent?

Reply:
I think we should remove `carma_ros2_utils::PubPtr<T>` as well. I'm not sure what the benefit is by aliasing the publisher type. `rclcpp::Publisher` and `rclcpp_lifecycle::LifecyclePublisher` have different interfaces and behaviors/expectations. If we want to be consistent, I suggest we use the `rclcpp` types instead of the `carma_ros2_utils` aliases. Thoughts?

Reply:
Yeah I think I agree. Changing back the aliases to its original syntax is not going to reduce the exception handling aspects of the carma, so I changed the ones in this package for the starters.